test/test_pmm.py

```diff
@@ -92,7 +92,8 @@ def setUp(self):
             order_amount=Decimal("1"),
             order_refresh_time=5.0,
             filled_order_delay=5.0,
-            order_refresh_tolerance_pct=-1
+            order_refresh_tolerance_pct=-1,
+            minimum_spread=-1,
```

Comment:
Make the default value 0. I can see a reason why users may want to make this parameter negative, so I think making the lower bound -1 is correct. However, I think users would want to default this at 0.

Reply:
I believe the default value is 0, and in this test case the value is -1 to disable the feature. However, the parameter allows for negative values in the UI, so I may need to consider a different way to disable the feature, or not allow non-negative numbers. A solution may be to add an additional parameter, such as "minimum_spread_enabled".